testing/square_bracket_accessor_test.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+#
+# square_bracket_accessor_test.py
+# Francois-Michel L'Heureux, 2017-01-27
+# This file is part of MLDB. Copyright 2017 mldb.ai inc. All rights reserved.
+#
+import unittest
+
+mldb = mldb_wrapper.wrap(mldb)  # noqa
+
+class Mldb2105SelectExprSquareBracketColNameSupport(MldbUnitTest):  # noqa
+
```

Comment:
First half of the test. Accessors with single quotes. This is what MLDB already does.